tempest/matrix/C44Matrix.hpp

```diff
@@ -53,4 +55,6 @@ C44Matrix operator*(const C44Matrix& l, float a);
 
 C44Matrix operator*(const C44Matrix& l, const C44Matrix& r);
 
+C4Vector operator*(const C4Vector& v, const C44Matrix& r);
```

Comment:
I'd prefer we keep global operators in the lhs class. ie, this `operator*` would go in `C4Vector.hpp`.